packages/sdk/test/integration/history_tree_concurrent_test.ts

```
@@ -0,0 +1,185 @@
+ * Convergence of concurrent OVERLAPPING undo/redo once the deleted nodes have
+ * been GC-purged (so restore takes the recreate path). The existing
+ * history_tree reconcile cases undo before GC runs and so never exercise this;
+ * these settle enough rounds to let GC purge the tombstones first. Regression
+ * for the multi-user tree undo corruption seen in wafflebase docs: split-aware
+ * restore/retombstone that isolates each piece at the span boundaries so all
+ * replicas converge on the same text-node segmentation.
+ */
+type TestDoc = { t: Tree };
+
+async function settle(c1: any, c2: any) {
+  for (let i = 0; i < 3; i++) {
```

ESLint

[error] Missing JSDoc comment.

(jsdoc/require-jsdoc)